### PR TITLE
Followup child cleanup PR

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
None

#### Purpose of change
I did some oopsies in #63311, primarily by missing some content that shouldn't spawn on children. This should hopefully address that.

#### Describe the solution

<details>
<summary>ITEMGROUP SHENANIGANS</summary>

- Both corset types should no longer spawn on children
- Nanoskirts should no longer spawn on children
- Catalytic reactor book (& connected note) no longer should spawn alongside anything related to children
- Glassblowing book has been removed from the textbook group. I have never heard of anyone teaching glassblowing at school. As a hobby, sure, but it doesn't fit a school textbook.
- Child's notes now have 3.3x the chance to spawn in their itemgroup
- The child loot group has been split - old ID left as is (only removed the bugout bags from it) while new one has a ``_pockets`` suffix - replaces the actual loot group on monsters. It is meant to only contain items kids would likely have been carrying on them as they turned. So no more weirdly strapped baseball bats.

</details>

<details>
<summary>ITEM SHENANIGANS</summary>

- Added a new fox plushie variant because Max asked me to
- Migrated old figurines into the toy version that supports variants. Truth be told, I didn't know we HAD figurines. If I did my approach would have been different in the initial PR. But now that ship's sailed so I'm stealing their snippets for more variant material
- Plush toys have been given ``NO_SALVAGE`` and a proper disassembly recipe. The weight of results is all over the place but that may be due to our handling of down filling. Anyway, this is because if you cut the plush toy right now you'd get hundreds upon hundreds of patches - the material does not account for down filling
- Some new child snippet notes because apparently people still want to make more

</details>

### CODE REVIEW

**Comment by Vaskritaya** [comment]
re: The figurines you're confused about: The first one is clearly just a barbie doll or any of the countless other generic girl's dolls. The one with a hat and clawed glove is probably freddy kreuger (hat, single bladed glove on right hand) but kreuger's glove has four blades, so who knows. 

The reindeer suit + skull one is a complete mystery to me. While searching, I did find [this amazon listing](https://www.amazon.com/Ebros-Christmas-Greetings-Furrybones-Collectible/dp/B0854PSWGJ). My theory is someone just searched "figurine" on amazon and used what they found to make some basic descriptions. This is further supported by [this listing that perfectly matches the unicorn figurines.](https://www.amazon.com/Summit-International-Unicorns-Figurines-Fantasy/dp/B07BJDR9T6/ref=sr_1_2?crid=2BFCJZ8SD9DM4&keywords=no+evil+unicorn+figurine&qid=1683422558&sprefix=no+evil+unicorn+figurine%2Caps%2C86&sr=8-2)

**Comment by Karol1223** [comment]
> The first one is clearly just a barbie doll or any of the countless other generic girl's dolls. 

I actually knew this one. The problem is... I don't think I can call it a barbie doll. I am 99% sure that's copyrighted.

> The one with a hat and clawed glove is probably freddy kreuger (hat, single bladed glove on right hand) but kreuger's glove has four blades, so who knows.

I thought so but I know nothing about Kreuger so I wanted confirmation, that's good to know I've not gone insane. Though I also will likely need a different name for it, I reckon. Legally distinct Freddy Kreuger.

> The reindeer suit + skull one is a complete mystery to me. While searching, I did find this amazon listing. My theory is someone just searched "figurine" on amazon and used what they found to make some basic descriptions. This is further supported by this listing that perfectly matches the unicorn figurines.

This makes sense but also is very problematic because this roughly translates to "what the fuck am I supposed to name them". But well, at least it's good to know they're generic figurines and not some obscure references.

**Comment by Ruminator24** [comment]
I'm not sure if you're still looking for more diary entries for https://docs.google.com/document/d/1RsZBQ9dXbbicVxwYyYI9IQ8Zk8uFgFBZI_OAThe9I88/edit
but i decided to add some, hopefully you won't mind? They're the newest ones I added.

File: data/json/items/toy.json
Comment (FoolsGold45) [inline data/json/items/toy.json:1]: superhero figurine
"A superhero action figure.  Its limbs can be adjusted for different poses, and it is designed to do battle against a wide variety of crooks and supervillains, sold separately."

**Review by FoolsGold45 — COMMENTED** [review]
As requested. Sorry about the weird comment formatting, doing this from mobile is hard.

**Review by FoolsGold45 — COMMENTED** [review]
As requested. Sorry about the weird comment formatting, doing this from mobile is hard.

**Comment by Karol1223** [comment]
Thank you a lot!

**Pull request merged**